### DISCUSSION TITLE
CutePDF

### DISCUSSION THREAD

**Discussion by BilleBarrett** [Devolutions/UniGetUI#616] [General]
For some reason CutePDF Writer will not update on any of my comupters
Log, is there anything I can do
-\       1024 KB / 11.9 MB     2.00 MB / 11.9 MB     3.00 MB / 11.9 MB     4.00 MB / 11.9 MB     5.00 MB / 11.9 MB     6.00 MB / 11.9 MB     7.00 MB / 11.9 MB     8.00 MB / 11.9 MB     9.00 MB / 11.9 MB     10.0 MB / 11.9 MB     11.0 MB / 11.9 MB     11.9 MB / 11.9 MB                                                                Found CutePDF Writer [AcroSoftware.CutePDFWriter] Version 4.0 This application is licensed to you by its owner. Microsoft is not responsible for, nor does it grant any licenses to, third-party packages. Successfully verified installer hash -  Starting package install... -\  Installer failed with exit code: 1 Installer log is available at: C:\Users\Chris\AppData\Local\Temp\WinGet\defaultState\WinGet-AcroSoftware.CutePDFWriter.4.0-2023-01-13-09-30-50.157.log

**Comment by marticliment** [comment]
could you please try updating using winget-cli? 

```
winget install -i AcroSoftware.CutePDFWriter -e
```

**Comment by BilleBarrett** [comment]
I will try it on the system that it didn’t work on
BTW – Great product

 

 

(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by BilleBarrett** [comment]
I did see an issue with Sandboxie-Plus, I’m a regular user of it and I have the latest release with is 1.6.5
I update my clients and I noticed that WingateUI tried to change it 1.6.4 
Just mentioning it

 

 

(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Reply by marticliment** [reply]
please know that those issues are winget fault, since winget is who is in charge of the actual updating of the packages.

**Comment by BilleBarrett** [comment]
Can you tell me why scoop tries to update and fails and I can’t stop it

 

 

(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Reply by marticliment** [reply]
honestly, try to do the same action on scoop cli, and if it still does not work, post on scoopinstaller
